WTG.Analyzers.Test/TestData/VisibilityAnalyzer/DetectAndFix/Diagnostics.xml

```diff
@@ -1,18 +1,19 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="utf-8" ?>
 <diagnostics id="WTG1001" message="Our convention is to omit the 'private' modifier where it is already the default." severity="Hidden">
+	<suppressId>CS0107</suppressId>
```

Comment:
Any chance we can use a newer build of Roslyn for the unit tests than the Analyzers themselves target?

Reply:
Maybe, but it would mean taking a dependency on two different versions of Roslyn ... not sure it's worth the added complexity.